eng/test-job.yml

```diff
@@ -323,6 +323,35 @@ jobs:
           - forcerelocs
           - gcstress15
 
+    # Zip tests
+    - ${{ if ne(parameters.corefxTests, true) }}:
+      - task: ArchiveFiles@2
+        displayName: Zip Tests
+        inputs:
+          rootFolderOrFile: $(Build.SourcesDirectory)/bin/tests/$(osGroup).$(archType).$(buildConfigUpper)
+
+          ${{ if ne(parameters.osGroup, 'Windows_NT') }}:
+            archiveType: tar
+            tarCompression: gz
+            archiveFile: $(Build.SourcesDirectory)/bin/tests/$(osGroup).$(archType).$(buildConfigUpper).tar.gz
+          ${{ if eq(parameters.osGroup, 'Windows_NT') }}:
+            archiveType: zip
+            archiveFile: $(Build.SourcesDirectory)/bin/tests/$(osGroup).$(archType).$(buildConfigUpper).zip
+          includeRootFolder: true 
+
+    - ${{ if ne(parameters.corefxTests, true) }}:
+      - task: PublishBuildArtifacts@1
+        displayName: Publish Test Build
+        inputs:
+          ${{ if ne(parameters.osGroup, 'Windows_NT') }}:
+            pathtoPublish: $(Build.SourcesDirectory)/bin/tests/$(osGroup).$(archType).$(buildConfigUpper).tar.gz
+          ${{ if eq(parameters.osGroup, 'Windows_NT') }}:
+            pathtoPublish: $(Build.SourcesDirectory)/bin/tests/$(osGroup).$(archType).$(buildConfigUpper).zip
+
+          ${{ if and(ne(parameters.corefxTests, true), eq(parameters.readyToRun, true)) }}:
+            artifactName: ${{ format('Tests_{0}_{1}_{2}_{3}_{4}_Logs', '_r2r',        parameters.osIdentifier, parameters.archType, parameters.buildConfig, parameters.testGroup) }}
+          ${{ if and(ne(parameters.corefxTests, true), ne(parameters.readyToRun, true)) }}:
+            artifactName: ${{ format('Tests_{0}_{1}_{2}_{3}_{4}_Logs', '',            parameters.osIdentifier, parameters.archType, parameters.buildConfig, parameters.testGroup) }}
```

Comment:
Maybe leave out the `_Logs` suffix. For consistency with the build artifacts, I'd suggest naming them `<osIdentifier>_<archType>_<buildConfig>_<testGroup>_tests`. Later it might be worth renaming all of the artifacts including logs for more consistency.

Reply:
I am in the middle of renaming these artifacts. See https://github.com/dotnet/coreclr/pull/26020